### PR TITLE
K8s cleaner

### PR DESCRIPTION
**Pull request by emamihe**
- added Dockerfile
- added CI/CD for building image and pushing into artifact registry

### CODE REVIEW

File: crates/k8s-cleaner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM rust:1.55 as builder
```
Comment (pepoviola) [inline crates/k8s-cleaner/Dockerfile:1]:
```suggestion
FROM rust:1.72 as builder
```

File: crates/k8s-cleaner/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM rust:1.72 as builder
+WORKDIR /usr/src/k8s-cleaner
+COPY . .
+RUN cargo build --release
+
+FROM debian:buster-slim
+WORKDIR /usr/local/bin
+COPY --from=builder /usr/src/k8s-cleaner/target/release/myrustapp k8s-cleaner
```
Comment (pepoviola) [inline crates/k8s-cleaner/Dockerfile:8]:
```suggestion
COPY --from=builder /usr/src/k8s-cleaner/target/release/k8s-cleaner k8s-cleaner
```

**Review by pepoviola — APPROVED** [review]

File: .github/workflows/k8s-cleaner.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    # - name: Install kubectl via gcloud
+    #   run: gcloud components install kubectl
+
+    # - name: Get GKE credentials
+    #   run: gcloud container clusters get-credentials $GKE_CLUSTER --zone $GKE_ZONE --project $PROJECT_ID
+
+    # - name: Set image in cronjob
+    #   env:
+    #     IMAGE: ${{ steps.build-image.outputs.image }}
+    #   run: kubectl set image cronjob/$GKE_CRONJOB $GKE_CONTAINER=$IMAGE -n $GKE_NAMESPACE
```
Comment (l0r1s) [inline .github/workflows/k8s-cleaner.yml:66]: Are these comments needed ?

**Review by l0r1s — APPROVED** [review]

File: .github/workflows/k8s-cleaner.yml (hunk 3 above)
Comment (pepoviola) [reply .github/workflows/k8s-cleaner.yml:66]: No, we can remove them and add some info about how to deploy a new image.

**Pull request merged**